app/handlers/handlers_connectors.py

```diff
@@ -47,6 +47,8 @@ def get_connector_pod(
     container_extra = spec.container_extra
     extra_env = container_extra.pop("env", [])
     extra_env_from = container_extra.pop("envFrom", [])
+    extra_volumes = container_extra.pop("volumes", [])
```

Comment:
`volumes` field should be specified in `pod_extra` , not `container_extra` like `volumeMounts` ([example](https://kubernetes.io/docs/concepts/storage/volumes/#configmap))